modular_skyrat/modules/conveyor_sorter/code/conveyor_sorter.dm

```diff
@@ -145,3 +149,43 @@
 		"conveysorter",
 	)
 	research_costs = list(TECHWEB_POINT_TYPE_GENERIC = 2500)
+
+/obj/item/conveyor_sorter/improved
+	name = "improved conveyor sorter lister"
+	desc = "A tool that is used to not only create the conveyor sorters, but give lists to the conveyor sorters."
+	icon_state = "lister_improved"
+	max_sorters = 8
+	max_items = 10
+	conveyor_type = /obj/effect/decal/cleanable/conveyor_sorter/improved
+
+/obj/effect/decal/cleanable/conveyor_sorter/improved
+	name = "improved conveyor sorter"
+	desc = "A mark that will sort items out based on what they are. This one can sort in multiple directions."
+	icon = 'modular_skyrat/modules/conveyor_sorter/icons/conveyor_sorter.dmi'
+	icon_state = "sorter_improved"
+	light_range = 3
+	light_color = COLOR_BLUE_LIGHT
+	directions = list("North", "East", "South", "West", "NorthEast", "NorthWest", "SouthEast", "SouthWest")
+
+/datum/design/conveyor_sorter/improved
+	name = "Improved Conveyor Sorter"
+	desc = "A wonderful item that can set markers and forcefully move stuff to a direction. With more capacity to sort more!"
+	id = "conveyor_sorter_improved"
+	build_path = /obj/item/conveyor_sorter/improved
+	materials = list(
+        /datum/material/iron = 500,
+        /datum/material/plastic = 500,
+        /datum/material/gold = 500,
+        /datum/material/bluespace = 500,
+    )
```

Comment:
This list feels weirdly indented, make sure that the entries in the list are indented one tab further than the word `materials`, and that the closing parenthesis is on the same indentation as `materials`.

Reply:
Just checked it is indeed indented correctly, but for some reason it had four spaces for the first indent and tab for the second

![image](https://user-images.githubusercontent.com/2568378/218272163-1a1da6c8-6eae-4b1b-9cd9-c783d77f8015.png)